<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moviePropertiesDialog</class>
 <widget class="QDialog" name="moviePropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Movie Resolution</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="resolutionCombo">
         <item>
          <property name="text">
           <string>320x240</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>640x480</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>800x600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024x768</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1280x1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1600x1200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Bit Rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="bitRate"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Frame Rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="frameRate"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="escButton">
         <property name="text">
          <string>Esc</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>released()</signal>
   <receiver>moviePropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>escButton</sender>
   <signal>released()</signal>
   <receiver>moviePropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
